RecoLocalTracker/SiPixelRecHits/src/SiPixelTemplateSplit.cc

```diff
@@ -47,7 +47,7 @@
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #define LOGERROR(x) edm::LogError(x)
 #define LOGDEBUG(x) LogDebug(x)
-static int theVerboseLevel = 2;
+static constexpr int theVerboseLevel = 2;
```

Comment:
Static guarantees the symbol is not exported. I've never checked to see if anonymous namespaces have the same guarantee.